Examples/SubscriberExample/Sources/Screens/Map/MapAnnotation/TruckAnnotationView.swift

```diff
@@ -1,7 +1,8 @@
 import Foundation
 import MapKit
 
-class TruckAnnotationView: MKAnnotationView {
+class TruckAnnotationView: MKAnnotationView, Identifiable {
```

Comment:
Truck? Was it heavy vehicle meant?)

Reply:
I've just added `Identifiable` conformance. This naming is not my idea.

Reply:
@KacperKluka WDYT? Is it always a truck? Or can be a bicycle?

Reply:
Well, the annotation image view is a truck and this is just an example app. Why do we want to force it to show a bicycle image? Is there any good reason?
So if we want to change the naming, I have to change the image too.